packages/repository-tests/src/types.repository-tests.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/repository-tests
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  Entity,
+  EntityCrudRepository,
+  juggler,
+  Options,
+} from '@loopback/repository';
+
+/**
+ * DataSource configuration (connector name, connection string, etc.).
+ */
+export type DataSourceOptions = Options;
+
+/**
+ * List of flags describing connector-specific behavior. These flags
+ * are used by the test suite to tweak assertions and skip tests
+ * for scenarios not supported by some connectors.
+ */
+export interface CrudConnectorFeatures {
```

Comment:
Just curious :) you use `interface` instead of `type` here, is it for the convenience of using declaration merging?

Reply:
When describing object properties, `interface` is the keyword to use. It allows other interfaces and classes to inherit from `CrudConnectorFeatures`. The keyword `type` does not allow inheritance, it defines a kind of a pointer/reference.